source/blog/2022-07-20-announcing-hanami-200beta1.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+title: Announcing Hanami v2.0.0.beta1
+date: 2022-07-20 11:04:32 UTC
+tags: announcements
+author: Luca Guidi
+image: true
+excerpt: >
+  Hanami 2 first beta: app-centric revolution & CLI. What to expect in 2.0 and 2.1.
+---
+
+Hello Hanami community! We’re thrilled to announce the release of Hanami 2.0.0.beta1!
+
+## App-centric revolution
+
+With this release, we’ve revolutionized the Hanami app structure: the **`app/` directory is now the primary home for your code, and **slices are now optional**.
+
+“What's a slice?,” you may ask! Think of slices as distinct modules of your application. A typical case is to use slices to separate your business domains (e.g. billing, accounting, admin). For our earlier 2.0 alpha releases, slices were the only way to build Hanami apps, which presupposed you wanted a full modular monolith composed of multiple domains.
+
+With this change, you can just as easily build small services with Hanami, with minimal ceremony.
+
+Let’s dissect a newly generated application (this one being named “Bookshelf”):
+
+- `app/` is home for your application code
+- `app/actions/` holds your actions and `app/action.rb`, with `Bookshelf::Action` as their superclass
+- `config/app.rb` defines your application (`Bookshelf::App`) and its configuration
+- `config/settings.rb` defines your app’s settings, such as the database URL
+- `config/routes.rb` defines your app’s HTTP routes
+- `lib/` is the home for any general purpose code used by your application code
+- `lib/bookshelf/types.rb` defines your custom Ruby data types (using the dry-types gem)
```

Comment:
You know, I wonder if could also go without generating a types module for basic Hanami apps...

From the perspective of getting started, it'd be useful for type-checking your settings (which is optional), but there's nothing else about a newly-generated Hanami app that uses the types.